packages/roosterjs-content-model-plugins/lib/paste/WordDesktop/processPastedContentFromWordDesktop.ts

```diff
@@ -71,7 +71,7 @@ function adjustPercentileLineHeight(format: ContentModelBlockFormat, element: HT
 function listLevelParser(
```

Comment:
since it is a parser, let's declare it as a parser, following the same pattern with line 84